scripts/upload_processor.py

```diff
@@ -0,0 +1,46 @@
+"""A function to process uploaded samples"""
```

Comment:
Can you give a broader overview of what this does, e.g. outline the overall steps?